lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+const getScriptFileName = require('../utilities/get-script-file-name');
+
+module.exports = function updateManifestFromHtml(root) {
+  const pkg = JSON.parse(fs.readFileSync(root + '/package.json', 'UTF8'));
+  const htmlFileName = pkg.fastboot.manifest.htmlFile || 'index.html';
+
+  // otherwise, we must parse the index.html file to figure out what's next
+  const indexHtml = fs.readFileSync(root + '/' + htmlFileName, 'UTF8')
+  // TODO: fix simple-html-tokenizer: https://github.com/tildeio/simple-html-tokenizer/pull/71
+    .replace('<!DOCTYPE html>','');
+
+  const assetsDir = `assets/`;
+  const vendorFiles = pkg.fastboot.manifest.vendorFiles || [];
+
+  // Find script src which are not data-fastboot-ignore
+  const indexScriptFilePaths = findScriptSrcs(indexHtml, findScriptSrcs.ignoreWithAttribute('data-fastboot-ignore'))
```

Comment:
Who will be adding `data-fastboot-ignore` attribute?

Reply:
Added this info to README and in code as well 
https://github.com/ember-fastboot/ember-cli-fastboot/blob/9ace0cd548a155d2ac92616ca0eb9acb97088423/README.md#update-manifest-from-html
https://github.com/ember-fastboot/ember-cli-fastboot/blob/9ace0cd548a155d2ac92616ca0eb9acb97088423/lib/embroider/update-manifest-from-html.js#L20